modules/cnv-creating-rbac-cloning-dvs.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-virtual-machines/cnv-cloning-vms/cnv-enabling-user-permissions-to-clone-datavolumes.adoc
+
+[id="cnv-creating-rbac-cloning-dvs_{context}"]
+= Creating RBAC resources for cloning DataVolumes
+
+Create a new ClusterRole that enables permissions for all actions for the `datavolumes` resource. 
+
+.Procedure
+
+. Create a ClusterRole manifest:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: <datavolume-cloner><1>
```

Comment:
For all callouts (numbery-pops :)) there should be a space between the code and the `<#>`